wallet/src/de/schildbach/wallet/ui/dashpay/notification/ContactViewHolder.kt

```diff
@@ -29,32 +28,44 @@ import de.schildbach.wallet.livedata.Status
 import de.schildbach.wallet.ui.dashpay.utils.display
 import de.schildbach.wallet.util.WalletUtils
 import de.schildbach.wallet_test.R
-import kotlinx.android.synthetic.main.notification_contact_request_received_row.view.*
+import de.schildbach.wallet_test.databinding.NotificationContactRequestReceivedRowBinding
 import org.dash.wallet.common.ui.avatar.ProfilePictureDisplay
 
-open class ContactViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
-        NotificationViewHolder(R.layout.notification_contact_request_received_row, inflater, parent) {
+open class ContactViewHolder(val binding: NotificationContactRequestReceivedRowBinding) :
+        NotificationViewHolder(binding.root) {
 
-    fun bind(notificationItem: NotificationItemContact, state: Resource<WorkInfo>?, isNew: Boolean,
+    fun bind(notificationItem: NotificationItemContact, state: Resource<WorkInfo>?, isNew: Boolean, isFirst: Boolean,
              recentlyModified: Boolean, showAvatar: Boolean,
-             onActionClickListener: OnContactActionClickListener? = null) {
+             acceptRequest: ((UsernameSearchResult, Int) -> Unit)?,
+             ignoreRequest: ((UsernameSearchResult, Int) -> Unit)?) {
 
         val usernameSearchResult = notificationItem.usernameSearchResult
 
-        itemView.apply {
-            setBackgroundResource(if (isNew) R.drawable.selectable_round_corners_white else R.drawable.selectable_round_corners)
+        binding.apply {
+            itemView.setBackgroundResource(
+                when {
+                    isNew && isFirst -> R.drawable.selectable_top_round_corners_light_blue_08
+                    isNew -> R.drawable.selectable_background_light_blue_08
+                    isFirst -> R.drawable.selectable_top_round_corners_dark
+                    else -> R.drawable.selectable_background_dark
+                }
+            )
+            val layoutParams = itemView.layoutParams as RecyclerView.LayoutParams
+            if (isFirst) {
+                layoutParams.bottomMargin = 0
+            } else {
+                layoutParams.topMargin = 0
+                layoutParams.bottomMargin = 0
+            }
```

Comment:
This section makes the first item in the list have rounded corners at the top along with a top margin from the resource.  Otherwise other items have square corners and no margin.  This way New items have a continuous blue background, rather than each item having its own rounded corner background.